recipes/gcc/all/conanfile.py

```diff
@@ -1,126 +1,193 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanException, ConanInvalidConfiguration
+from conan import ConanFile, conan_version
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.layout import basic_layout
+from conan.tools.apple import XCRun
+from conan.tools.files import copy, get, replace_in_file, rmdir, rm
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc
 import os
 
-required_conan_version = ">=1.29.1"
+required_conan_version = ">=1.53.0"
 
 
 class GccConan(ConanFile):
     name = "gcc"
-    description = "The GNU Compiler Collection includes front ends for C, " \
-                  "C++, Objective-C, Fortran, Ada, Go, and D, as well as " \
-                  "libraries for these languages (libstdc++,...). "
+    description = (
```

Comment:
Consider adding package_type as recommended in Conan 2.0 documentation (see https://docs.conan.io/en/2.0/reference/conanfile/attributes.html?highlight=package_type#package-type)

>     package_type = "application"

Reply:
I'd thought about this, but I think that there's also utility in exporting the libraries as well. Especially as it relates to `gfortran`, `libgfortran` is often required at link time for any generated executable, so I'm not sure it's feasible for this to be considered only an `application` package.